source/jormungandr/jormungandr/parking_space_availability/bss/atos.py

```diff
@@ -68,18 +68,27 @@ def get_informations(self, poi):
         try:
             all_stands = self.breaker.call(self._get_all_stands)
             ref = poi.get('properties', {}).get('ref')
-            if ref:
-                stands = all_stands.get(ref.lstrip('0'))
+            if not ref:
+                return Stands(0, 0, 'Unavailable')
+            stands = all_stands.get(ref.lstrip('0'))
+            if stands:
+                if stands.status != 'Open':
```

Comment:
Shouldn't we move that behavior to genericize it?
It's not specific to Atos, no?